### PR TITLE
refactor and polish the code

### PR DESCRIPTION
**Pull request by zhiyelee**
refactor the code and polish log

### CODE REVIEW

File: lib/github.js [hunk 1]
```diff
@@ -0,0 +1,178 @@
+
```
Comment (yuhaoju) [inline lib/github.js:1]: Remove this line.

File: lib/github.js [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+    if (state === 'CHANGES_REQUESTED') changesRequestedCount += 1;
+    if (state === 'APPROVED') approvalCount += 1;
+  });
+
+  return {
+    changesRequestedCount,
+    approvalCount,
+    requiredApprovalCount,
+  };
+}
```
Comment (yuhaoju) [inline lib/github.js:178]:
```suggestion
}

```

**Review by yuhaoju — APPROVED** [review]

File: lib/github.js (hunk 1 above)
Comment (zhiyelee) [reply lib/github.js:1]: good catch

File: lib/github.js (hunk 2 above)
Comment (zhiyelee) [reply lib/github.js:178]: I think I need to tune my vscode config

**Pull request merged**